Special abilities PR 5, Split Speed values (#8)
GameBrains/er-core: modules/main-rules/pages/assault-moves-and-charges.adoc

And where you do move any individual unit into base contact with an enemy unit, we call this a *charge move*.

* You don't have to declare and make a charge into close combat unless you want to, with one exception: If the 'must move towards the enemy' and 'must move at least 5 cm, or into base contact' requirements force you to place units in base contact, then you have made charge moves by definition and so you have a charge whether you like it or not!
Of course you may move more units into base contact, where they can reach.
* If you don't get at least one unit into base contact with an enemy then there are no charge moves by definition, which means that the detachment does not charge and there will be no close combat.
(Though there will be a firefight if any units are within 15cm of the enemy.)

You can benefit from a special increase to the Speed value of some particular units when you make a charge move with them.
(Remember, it is only a charge move if the units will get into base contact with the enemy -- you don't get the increase if they won't get into base contact.)

* You may double the Speed value of any *Infantry* unit.

> * Where a unit has a split Speed value such as `20/30 cm`, you may *not* double the first value -- use the second value instead.
> * For any other unit, must use their Speed value in the usual way.

Author: This is ambiguous, and can be read as doubling the second value instead of doubling the first value.

I've also removed an errant 'must' from the next bullet.

```suggestion
* Where a unit has a split Speed value such as `20/30 cm`, use the second value *instead* of doubling the first value.
* For any other unit, use their Speed value in the usual way.
```

Reviewer: Well spotted, I agree that it could be read that way, and indeed there is an errant 'must'.
Thinking about it, I don't think we really need to mention "don't double" because that only applies to the previous bullet anyway — strictly, we don't use the second value instead of double-Speed (which we wouldn't do anyway for Cavalry because they count as Vehicle not Infantry). Rather, we use the second value instead of the normal Speed (first value), as one of a couple of special cases for a charge move .
```suggestion
* Where a unit has a split Speed value such as `20/30 cm`, use the second value.
* For any other unit, use their Speed value in the usual way.
```

Author: Don't forget that this covers Jump Packs units as well, which as Infantry **do** default to getting double Speed. 

Maybe:

```suggestion
* Where a unit has a split Speed value such as `20/30 cm`, use the second value but you cannot double it.
* For any other unit, use their Speed value in the usual way.
```

Reviewer: Balls. Of course, the preceding bullet doesn't work for Infantry that have Jump Packs. I'll add a new suggestion which covers all the lines together.

Reviewer: I'll resolve this conversation, as we have a new conversation on all three bullet rather than just the latter two.

> * You may double the Speed value of any *Infantry* unit.
> * Where a unit has a split Speed value such as `20/30 cm`, you may *not* double the first value -- use the second value instead.
> * For any other unit, must use their Speed value in the usual way.

Reviewer: ```suggestion
* You may double the Speed value of any *Infantry* unit, unless it has a split Speed value.
* Where a unit has a split Speed value such as `20/30 cm`, use the second value.
* For any other unit, use its Speed value in the usual way.
```

Author: Looks good to me.

Reviewer: Great, I'll commit it and resolve. This is nice, we're definitely honing things between us to be much more clear, complete and accurate than our first passes.

* You don't have to declare and make a charge into close combat unless you want to, with one exception: If you can only meet the 'must move towards the enemy' requirement by moving toward an enemy that has units that are within 5cm of your units, then you must charge and place units in base contact.
* For a charge, you may double the Speed value of *infantry* units *if* this will bring them into base contact with enemy units.
* If you don't get at least one unit into base contact with an enemy then there is no charge and there will be no close combat.
(Though there will be a firefight if any units are within 15cm of the enemy.)

> * Where a unit has a split Speed value such as 20/30 cm, you may use the second value if this would bring the unit into contact with an enemy. You *cannot* double the Speed value of the unit during an Assault move. +

Author: Rewrite of the split Speed text to fit better.

```suggestion
* For units with a split Speed value such as 20/30 cm, you may use the second value for a charge *if* this will bring them into base contact with enemy units. This is *instead* of doubling the units' Speed value. +
```

Reviewer: Will resolve conversation, as outdate - newer commit has fixed this.
